rcl/CMakeLists.txt

```diff
@@ -81,6 +85,9 @@ ament_target_dependencies(${PROJECT_NAME}
   ${RCL_LOGGING_IMPL}
   "rosidl_runtime_c"
   "tracetools"
+  "rosidl_typesupport_introspection_c"
+  "rosidl_typesupport_c"
+  "service_msgs"
```

Comment:
I _think_ the order here doesn't matter here, so I would alphabetize. I think the comment at L77 is vestigial and could be removed.